src/metabase/models/persisted_info.clj

```diff
@@ -137,12 +136,13 @@
 (defn ready-unpersisted-models!
   "Looks for all new models in database and creates a persisted-info ready to be synced."
   [database-id]
-  (let [cards (t2/select Card {:where [:and
-                                       [:= :database_id database-id]
-                                       [:= :dataset true]
-                                       [:not [:exists {:select [1]
-                                                       :from [:persisted_info]
-                                                       :where [:= :persisted_info.card_id :report_card.id]}]]]})]
+  (let [cards (t2/select :model/Card
```

Comment:
A bunch of `Model` -> `:model/Model` sort of changes were added to avoid circular dependencies that might be gone now. But it's still worth changing these afterall, right?